sandpack-react/package.json

```diff
@@ -20,8 +20,7 @@
     "prebuild": "yarn run clean",
     "test": "TEST_ENV=true jest . --transformIgnorePatterns \"node_modules/(?!@codemirror)/\" --modulePathIgnorePatterns \"e2e\"",
     "lint": "eslint '**/*.ts?(x)' --fix",
-    "build": "rollup -c && yarn build:types",
-    "build:types": "tsc -p tsconfig.json",
+    "build": "rollup -c --bundleConfigAsCjs",
```

Comment:
We're making this very same change on https://github.com/codesandbox/sandpack/pull/798

Reply:
Feel free to contribute there too